Sources/MongoSwift/ChangeStreamDocument.swift

```diff
@@ -0,0 +1,67 @@
+/// An `UpdateDescription` containing fields that will be present in
+/// the change stream document for operations of type `update`.
+public struct UpdateDescription: Codable {
+   /// A document containing key:value pairs of names of the fields
+   /// that were changed, and the new value for those fields.
+   public let updatedFields: Document
+
+   /// An array of field names that were removed from the document.
+   public let removedFields: [String]
+}
+
+/// An enum representing the type of operation for this change.
+public enum OperationType: String, Codable {
+   case insert
+   case update
+   case replace
+   case delete
+   case invalidate
+   case drop
+   case dropDatabase
+   case rename
+}
+
+/// The response document type from a `ChangeStream`.
+public struct ChangeStreamDocument<T: Codable>: Codable {
+    /// Describes the type of operation for this change.
+    public let operationType: OperationType
+
+    /// An opaque token for use when resuming an interrupted change
+    /// stream.
+    public let _id: ChangeStreamToken
+
+    /// A document containing the database and collection names in
+    /// which this change happened.
+    public let ns: MongoNamespace
+
+    /** Only present for options of type ‘insert’, ‘update’,
```

Comment:
This line should be moved one down so that the `/**` is on its own line.

Also, most of the documentation in this file can be expanded to the full column limit.